README.md

```diff
@@ -21,47 +22,49 @@ we store those as well:
 
 ```bash
 buildtest/tools/buildsystem/schemas
-   singlesource
-   singlecompile
+   script
 ```
 
+The above would include a folder of schemas for the "script" type, each named by a version.
+For example:
+
+
+```bash
+script/
+    script-v0.0.1.schema.json
+    script-v0.0.2.schema.json
+    ...
+```
+
+
 ## How is this repository organized?
 
 The entire root of this folder could be added to the folder shown above,
 and the subfolder structure would be added to the module. Importantly,
 the lowercase class names should match to a class defined in in [buildtest/tools/buildsystem/bases.py]().
```

Comment:
minor typo **class defined in in** --> **class defined in**

Reply:
Fixed locally.